test/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 # Enable the tests
 
+# Allow the source files to find headers in src/
+include_directories(${PROJECT_SOURCE_DIR}/src)
```

Comment:
see https://github.com/google/benchmark/pull/199

the benchmark test there uses std::this_thread::sleep_for which doesn't require pulling from src.